src/pages/index.js

```diff
@@ -96,6 +92,17 @@ const profileEditModalPopup = new PopupWithForm(
   (data) => {
     currentUserInfo.setUserInfo(data.title, data.description);
     profileEditModalPopup.close();
+    api.editCurrentUser("/users/me", data);
+  }
+);
+
+const profileAvatarEditModalPopup = new PopupWithForm(
+  "#modal_avatar-edit",
+  (data) => {
+    profileAvatarEditModalPopup.close();
+    api.editUserAvatar("/users/me/avatar", data).then((res) => {
+      profileAvatar.src = res.avatar;
```

Comment:
Needs correction: please implement the method for updating the avatar in user info class and use it here and when initializing the user

Reply:
Done